### PR TITLE
chase renames NFC

### PR DESCRIPTION
**Pull request by amilcarlucas**
I really like explicit code

### CODE REVIEW

**Comment by amilcarlucas** [comment]
Regarding Travis, the build tests pass, but the SITL simulation gets stuck. It might be just a unrelated race condition.

**Comment by amilcarlucas** [comment]
Any news on this ??

**Comment by amilcarlucas** [comment]
Can someone please re-trigger travis ?

File: libraries/AP_Follow/AP_Follow.cpp [hunk 1]
```diff
@@ -186,14 +186,14 @@ bool AP_Follow::get_target_dist_and_vel_ned(Vector3f &dist, Vector3f &dist_with_
...
-    vel = veh_vel;
+    dist_ned_m = dist_vec;
+    dist_with_offs_ned_m = dist_vec + offsets;
+    vel_ned_ms = veh_vel;
     return true;
 }
 
 // get target's heading in degrees (0 = north, 90 = east)
-bool AP_Follow::get_target_heading(float &heading) const
+bool AP_Follow::get_target_heading_deg(float &heading) const
```
Comment (peterbarker) [inline libraries/AP_Follow/AP_Follow.cpp]: Again, shouldn't we assume degrees for a heading?

File: libraries/AC_Avoidance/AC_Avoid.h [hunk 2]
```diff
@@ -87,29 +87,29 @@ class AC_Avoid {
...
      * Adjusts the desired velocity for the beacon fence.
      */
-    void adjust_velocity_beacon_fence(float kP, float accel_cmss, Vector2f &desired_vel, float dt);
+    void adjust_velocity_beacon_fence(float kP, float accel_cmss, Vector2f &desired_vel_cms, float dt);
 
     /*
      * Adjusts the desired velocity based on output from the proximity sensor
      */
-    void adjust_velocity_proximity(float kP, float accel_cmss, Vector2f &desired_vel, float dt);
+    void adjust_velocity_proximity(float kP, float accel_cmss, Vector2f &desired_vel_cms, float dt);
```
Comment (peterbarker) [inline libraries/AC_Avoidance/AC_Avoid.h]: This is a good change.

**Comment by amilcarlucas** [comment]
Now that it has been merged to master I added this to ardupilot/ardupilot#7845

**Pull request closed without merging**